x/ibc/02-client/types/height.go

```diff
@@ -132,8 +132,73 @@ func ParseHeight(heightStr string) (Height, error) {
 	return NewHeight(epochNumber, epochHeight), nil
 }
 
+// SetEpochNumber takes a chainID in valid epoch format and swaps the epoch number
+// with the given epoch number.
+func SetEpochNumber(chainID string, epoch uint64) (string, error) {
+	if !IsEpochFormat(chainID) {
+		return "", sdkerrors.Wrapf(
+			sdkerrors.ErrInvalidChainID, "chainID is not in epoch format: %s", chainID,
+		)
+	}
+
+	splitStr := strings.Split(chainID, "-")
+	// swap out epoch number with given epoch
+	splitStr[len(splitStr)-1] = fmt.Sprintf("%d", epoch)
+	return strings.Join(splitStr, "-"), nil
+}
+
+// IsEpochFormat is a utility function that returns whether the given ChainID
+// is in the epoch format `{chainID}-epoch-{epochNumber}`
```

Comment:
cc: @cwgoes

Reply:
Sure; let's do that, I'll update the spec as well - https://github.com/cosmos/ics/issues/481

Reply:
@AdityaSripal I think we want to rename more than just the comment but the entire struct and funcs as well. This can be done in a followup, but we should open an issue to track it if so

Reply:
This should be done in a followup if we decide on this